package.xml

```diff
@@ -143,6 +143,22 @@
  <providesextension>ast</providesextension>
  <extsrcrelease />
  <changelog>
+  <release>
+   <date>2021-11-27</date>
+   <version>
+    <release>1.0.15</release>
+    <api>1.0.15</api>
+   </version>
+   <stability>
+    <release>devel</release>
+    <api>stable</api>
+   </stability>
+   <license uri="https://github.com/nikic/php-ast/blob/master/LICENSE">BSD-3-Clause</license>
+   <notes>
+- Allow ast\Node to have dynamic properties without emitting a notice in PHP 8.2.
+- Document that ast\Node implements `#[AllowDynamicProperties]` in php 8.2
```

Comment:
This part isn't in 1.0.15?

Reply:
Fixed, 1.0.15 was approved while I was working on updating the stubs and package.xml and I copied it as is